modules/mobile/app/models/mobile/v0/authorized_services.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class AuthorizedServices
+      def initialize(user)
+        @user = user
+      end
+
+      def to_a
+        authorized_services_map.select { |_, v| v == true }.keys
+      end
+
+      def available_services
+        authorized_services_map.keys
+      end
+
+      private
+
+      def authorized_services_map
+        {
+          appeals: access?(appeals: :access?),
```

Comment:
Currently, claims and appeals does not check this. Jayson and I discussed creating a ticket for making the controller use both access policies.